docs/core/about.md

```diff
@@ -0,0 +1,116 @@
+---
+title: About .NET Core
+description: .NET Core is a modular, high-performance implementation of .NET for creating Windows, Linux, and Mac apps. Learn about .NET Core to get started.
```

Comment:
The description here should probably be different from the other topic and tell what users will find in this page